packages/eas-json/src/types.ts

```diff
@@ -21,6 +21,7 @@ export interface EasJson {
     version?: string;
     requireCommit?: boolean;
     appVersionSource?: AppVersionSource;
+    setUpPushNotifications?: boolean;
```

Comment:
```suggestion
    promptPushNotficationsSetup?: boolean;
```
or
```suggestion
    promptToConfigurePushNotfications?: boolean;
```
setUpPushNotifications might suggest that setting it to false will disable push ntifications

Reply:
@brentvatne ?

Reply:
my vote is for `promptToConfigurePushNotifications`

Reply:
`promptToConfigurePushNotfications` sounds good to me too!